samples/Azure.AI.FormRecognizer/Generated/AzureAIFormRecognizerRestClient.cs

```diff
@@ -303,9 +303,7 @@ internal HttpMessage CreateAnalyzeWithCustomModelRequest(Guid modelId, bool? inc
             if (fileStream != null)
             {
                 request.Headers.Add("Content-Type", "application/json");
-                var content = new Utf8JsonRequestContent();
```

Comment:
There is a problem in the original swagger definition: https://github.com/Azure/azure-rest-api-specs/blob/e606243e5297312781dd7dbfd7ab76d2329cc088/specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json#L171-L177

Reply:
actually, it's a bug in the change